docs/schemas/annotations.md

```diff
@@ -0,0 +1,157 @@
+    attributes:
+      id: ...
+```
+
+This will result in an error, because `review` is not a metamodel slot.
+
+This conformance to a single metamodel can be useful for ensuring consistency and interoperability
+among LinkML schemas. However, it can be too constraining in some cases. To get around this,
+the LinkML metamodel has a generic [annotations](https://w3id.org/linkml/annotations) slot that can be used
+to assign arbitrary tags and values to any schema element.
+
+## Adding your own annotations
+
+In the above case, we can add the `review` annotation to the `Person` class like this:
+
+```yaml
+classes:
+  Person:
+    annotations:
+      review: A very useful class that is well defined
+      ...
+```
+
+Note that in the underlying metamodel, the range of [annotations](https://w3id.org/linkml/annotations) is
+an [Annotation](https://w3id.org/linkml/Annotation) class, which has two slots: `tag` and `value`. This is
+usually written in *compact form* as `tag: value`, as in the above.
+
+Note also that prior to LinkML 1.6, the range of the annotation value was `string`, but in 1.6 and higher,
+the range can be any object.
+
+## Annotation validation
+
+By default, all annotation tags and values are valid. Feel free to add whatever annotations you like
+to any schema element - classes, enums, permissible values, slots, even schemas themselves.
+
+However, if you want to restrict the set of valid annotations, you can do this starting from LinkMK 1.6,
+where you can treat annotation tags as if they were slots in your own metamodel extension.
+
+This is done by adding an [instantiates](https://w3id.org/linkml/instantiates) slot-value assignment onto any schema element. The range of
+the `instantiates` slot is a `uriorcurie` that references a class that serves as a metamodel extension class.
+
+For example, given our example schema fragment, we can add an `instantiates` slot to the `Person` class,
+stating that this class is an instance of the `Reviewable` metamodel extension class:
+
+```yaml
+classes:
+  Person:
+    instantiates: 
+      - mymetamodel:Reviewable
+    annotations:
+      review: A very useful class that is well defined
+      ...
+```
+
+This metaclass extension can live in the same schema, or in a different schema. This is why they are referenced
+by `uriorcurie` rather than a local name.
+
+The `Reviewable` class might be defined as follows:
+
+```yaml
+id: https://example.org/mymetamodel/
+name: mymetamodel
+...
+classes:
+  Reviewable:
+    class_uri: mymetamodel:Reviewable
+    attributes:
+      review:
+        description: an expert review of a schema element
+        range: string
+```
+
+This can be thought of as a schema element mixin class - the class defines the extra slots that can be
+attached to schema elements via annotations.
+
+Other schema elements can also instantiate the `Reviewable` class:
+
+```yaml
+classes:
+  Person:
+    instantiates: 
+      - mymetamodel:Reviewable
+    annotations:
```

Comment:
how does this `annotations` component relate to the Reviewable class? 
if I think of it like a mixin, then I would expect to only see `annotations` in the `attributes` section?

Reply:
Added a section:

Note that here the `attributes` section of the `Person` definition is the attributes that an
instance of `Person` may have. The `annotations` section pertains to the *class* Person, not instances.